routes/applications.ts

```diff
@@ -24,6 +24,5 @@ router.patch(
   applicationValidator.validateApplicationUpdateData,
   applications.updateApplication
 );
-router.patch("/batch/update", authenticate, authorizeRoles([SUPERUSER]), applications.batchUpdateApplications);
```

Comment:
Did you check when we created application createdAt and the status is successfully added?

Reply:
no, let me check

Reply:
yep they are added when a new Application is added, also, the status is hardcoded as pending

Reply:
by tracing the codebase, How do I get the rds-session token to hit the POST applications?

Reply:
First, run the local backend and in the browser run ```http://localhost:3000/auth/github/login``` and authorize the GitHub Then in cookie you will the rds-session-development  token